src/sysrepo.c

```diff
@@ -3462,6 +3462,18 @@ sr_has_changes(sr_session_ctx_t *session)
     return 0;
 }
 
+API int
```

Comment:
Would probably just return `const struct lyd_node *`, the error is not interesting.